examples/reactnavigation/src/DrawerAppNavigator.js

```diff
@@ -0,0 +1,38 @@
+import 'react-native-gesture-handler';
```

Comment:
This must be done in the entry file

Reply:
Nice catch!
I moved the import to the entry file. I also moved the import in the Stack Navigator example file, because it was in the same place.
(I copied the stack navigator example files and changed the navigation for the drawer)